pkg/shipyard/logger.go

```diff
@@ -0,0 +1,46 @@
+package shipyard
+
+import (
+	"log"
+	"strings"
+)
+
+var Log Logger = &StandardLogger{}
```

Comment:
Is there a particular reason why you favour your custom logger implementation instead of using micrologger?

Reply:
Not at all :) moving to micrologger